datasets/modeled/transport/model_year_to_model_year.csv

```diff
@@ -0,0 +1,11 @@
+from_id,to_id,from_fraction
+2025,2024,1
```

Comment:
It seems like this should be a fraction a little less than one? Maybe you can estimate by looking at historic EV sales for 2021, 2022, and 2023?

Reply:
Agreed, I'll look into a better number here and for the between year interpolations and possibility of DECARB 2 input.